test/extensions/filters/http/cdn_loop/parser_fuzz_test.cc

```diff
@@ -0,0 +1,19 @@
+#include "extensions/filters/http/cdn_loop/parser.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  {
```

Comment:
do you expect more of the API to be fuzzed? (if so, makes sense to keep these brackets)

Reply:
I removed them.